internal/pkg/debug/render/ssa.go

```diff
@@ -0,0 +1,41 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package render
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/google/go-flow-levee/internal/pkg/debug/node"
+	"golang.org/x/tools/go/ssa"
+)
+
+// SSA produces a human-readable representation of the SSA code for a function.
+func SSA(f *ssa.Function) string {
```

Comment:
WDYT about creating a wrapper type for ssa.Function?

This type could then implement Stringer interface, and ToDOT method.

Reply:
I'm not sure how I would call such a type. The name `SSAFunction` comes to mind, but it seems redundant somehow. Just `Function` might do, but I don't find it sufficiently evocative.

I'm also not sure that having `String()` for SSA and a separate function for generating DOT source would be super intuitive. Maybe have 2 wrapper types, one for SSA and one for DOT, each with its own `String()` method? I could also create a wrapper type for `ssa.Instr`.

I think it would be best for me to explore this in a future PR. WDYT?